cypress/support/commands.js

```diff
@@ -295,13 +295,13 @@ Cypress.Commands.add('generateLabResults', (uuid) => {
     });
 })
 
-Cypress.Commands.add('createPatient', (identifier = null) => {
+Cypress.Commands.add('createPatient', (identifier = null, name) => {
```

Comment:
Let's make it more readable. (remove 'john' from line 304)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

```suggestion
Cypress.Commands.add('createPatient', (identifier = null, name = 'John') => {
```